packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroup.types.ts

```diff
@@ -44,6 +45,16 @@ export interface IChoiceGroupProps extends React.InputHTMLAttributes<HTMLElement
    * @deprecated
    */
   onChanged?: (option: IChoiceGroupOption, evt?: React.FormEvent<HTMLElement | HTMLInputElement>) => void;
+
+  /**
+   * Theme (provided through customization.)
+   */
+  theme?: ITheme;
+
+  /**
+   * Call to provide customized styling that will layeron top of the variant rules.
+   */
+  getStyles?: IStyleFunction<IChoiceGroupStyleProps, IChoiceGroupStyles>;
```

Comment:
>getStyles? [](start = 2, length = 10)

Since you are introducing it here, it would be nice to have an example around its usage in the example pages we have for ChoiceGroup.

Reply:
In 6.0 we're decided to stick with "styles" as the prop name, and allow it to support a function or simple style object as a value. Can we get inline with that approach?